data/brands/amenity/bank.json

```diff
@@ -13520,20 +13598,20 @@
       }
     },
     {
-      "displayName": "東亞銀行",
+      "displayName": "東亞銀行 Bank of East Asia",
       "id": "bankofeastasia-442838",
       "locationSet": {"include": ["hk"]},
       "matchNames": ["bea", "東亞"],
       "tags": {
         "amenity": "bank",
-        "brand": "東亞銀行",
+        "brand": "東亞銀行 Bank of East Asia",
         "brand:en": "Bank of East Asia",
         "brand:wikidata": "Q806679",
         "brand:wikipedia": "zh:東亞銀行",
         "brand:zh-Hans": "东亚银行",
         "brand:zh-Hant": "東亞銀行",
         "name": "東亞銀行 BEA",
-        "name:en": "Bank of East Asia",
+        "name:en": "BEA",
         "name:zh-Hans": "东亚银行",
```

Comment:
Will `short_name=BEA` better? And it can be match.

Reply:
Yes, we can add as `short_name`. 
Did `short_name` also have bilingual tags? Like `short_name:en` `short_name:zh` something like that.

Reply:
> `short_name:en` `short_name:zh`

Yes, we have this kind of usage:

https://github.com/osmlab/name-suggestion-index/blob/cb5996bbb9f84c71abfc849e4c73d9a5a3dc1211/data/brands/shop/supermarket.json#L9198-L9203